diff --git a/hardware/opentrons_hardware/hardware_control/move_group_runner.py b/hardware/opentrons_hardware/hardware_control/move_group_runner.py
index d085fbbddb..4e478b1f9d 100644
--- a/hardware/opentrons_hardware/hardware_control/move_group_runner.py
+++ b/hardware/opentrons_hardware/hardware_control/move_group_runner.py
@@ -394,7 +394,7 @@ class MoveScheduler:
         log.error(f"Error during move group: {message}")
         if severity == ErrorSeverity.unrecoverable:
             self._should_stop = True
-        self._event.set()
+            self._event.set()
 
     def _handle_move_completed(
         self, message: _AcceptableMoves, arbitration_id: ArbitrationId
