diff --git a/api/src/opentrons/config/feature_flags.py b/api/src/opentrons/config/feature_flags.py
index 583dae0b14..a6c2fff7e1 100644
--- a/api/src/opentrons/config/feature_flags.py
+++ b/api/src/opentrons/config/feature_flags.py
@@ -67,6 +67,4 @@ def status_bar_enabled() -> bool:
 
 def require_estop() -> bool:
     """Whether the OT3 should allow gantry movements with no Estop plugged in."""
-    return not advs.get_setting_with_env_overload(
-        "estopNotRequired", RobotTypeEnum.FLEX
-    )
+    return False
diff --git a/hardware/opentrons_hardware/hardware_control/motion_planning/move_utils.py b/hardware/opentrons_hardware/hardware_control/motion_planning/move_utils.py
index 8792d9fb59..0119278a39 100644
--- a/hardware/opentrons_hardware/hardware_control/motion_planning/move_utils.py
+++ b/hardware/opentrons_hardware/hardware_control/motion_planning/move_utils.py
@@ -24,7 +24,7 @@ log = logging.getLogger(__name__)
 
 FLOAT_THRESHOLD = 0.001  # TODO: re-evaluate this value based on system limitations
 
-MINIMUM_DISPLACEMENT = 0.05
+MINIMUM_DISPLACEMENT = 0.01
 
 # Minimum vector component of 0.1%
 MINIMUM_VECTOR_COMPONENT = np.float64(0.001)
